app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -227,7 +227,23 @@ class BrowserToolbarView(
 
     fun expand() {
         if (!settings.shouldUseBottomToolbar) {
-            layout.app_bar?.setExpanded(true)
+            view.app_bar?.setExpanded(true)
```

Comment:
I don't think this will work because `app_bar` contains `view`: I think you need `layout.app_bar` or maybe even just `app_bar`.

Reply:
@mcomella Hm the receiver type it is expecting is View.app_bar. The previous private val layout that they used here was also a view (since it was the return value of LayoutInflater.inflate). Does that mean it was incorrect in the first place?